PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -134,8 +134,9 @@
 # --- 
 
 _pp_on_AA_extraCommands = [
-    'keep patPackedCandidates_hiPixelTracks_*_*',
-	  'keep *_packedCandidateMuonID_*_*',
+    'keep patPackedCandidates_hiPixelTracks_*_*'
```

Comment:
@mandrenguyen a few commas are missing here
While fixing it, please also adjust the indentation